demos/python_demos/multi_camera_multi_person_tracking/requirements.txt

```diff
@@ -1,3 +1,19 @@
-opencv-python
-numpy
-scipy
+Cython==0.29.12
+setuptools
+gdown==3.8.3
+glog==0.3.1
+h5py==2.9.0
+lxml==4.3.4
+matplotlib==3.0.3
+motmetrics==1.1.3
+pandas==0.24.2
+opencv-python==3.4.5.20
+Pillow==6.1.0
+pycocotools==2.0.0
+requests==2.22.0
+numpy==1.16.4
+scipy==1.3.0
+six==1.12.0
+tensorboard==1.8
+tqdm==4.32.2
+yacs==0.1.6
```

Comment:
This file should not pin versions. If you have certain minimum requirements, you can use `>=`, but otherwise the demos are expected to work with any recent versions of dependencies.

Reply:
Fixed, used `>=`. I did not try early versions and it is possible that something turned out incompatible.